.github/workflows/ci.yml

```
@@ -104,6 +104,7 @@ jobs:
         run: flake8
       - name: Mypy
         run: mypy .
+        if: !matrix.numfocus_nightly
```

Github actions is complaining about the syntax
Maybe this is worth trying?
```suggestion
        if: ${{ !matrix.numfocus_nightly }}
```